tests/test_mysql.py

```diff
@@ -24,6 +24,10 @@ def test_connection():
     wr.mysql.connect("aws-data-wrangler-mysql", connect_timeout=10).close()
 
 
+def test_connection_ssl():
+    wr.mysql.connect("aws-data-wrangler-mysql-ssl", connect_timeout=10).close()
```

Comment:
Do you think we should be not only testing the ssl connection, but also use that connection for some of the mysql test queries? Was wondering if we could adapt the current pytest [fixture](https://github.com/awslabs/aws-data-wrangler/blob/2da313473e6426128fff0111ae819fb55ccc87ee/tests/test_mysql.py#L16) to take ssl enabled as an argument.

Something along the lines of:
```

@pytest.fixture(scope="function")
def mysql_con(ssl_enabled: bool = False):
    con_str = "aws-data-wrangler-mysql-ssl" if ssl_enabled else "aws-data-wrangler-mysql"
    con = wr.mysql.connect(con_str)
    yield con
    con.close()
```

Reply:
Agreed, it's an overkill. One or two tests should be enough!